src/main/java/org/catmaid/ScaleCATMAID.java

```diff
@@ -72,46 +72,55 @@
 	{
 		public int tileWidth;
 		public int tileHeight;
+		public long minC;
```

Comment:
I thought this would make it possible to parallelize by x and y but I can see why the entire z-section needs to be available in order for the scaling to be acurate.